Dockerfile

```diff
@@ -42,7 +48,8 @@ COPY --from=gopher /go/bin/protoc-gen-go-grpc /usr/bin/
 COPY --from=java /installer/protoc-gen-grpc-java /usr/bin/
 COPY --from=node /node_modules /node_modules
 COPY --from=protoc-bin /installer/protoc /usr/bin/protoc
-
+COPY --from=protoc-bin /installer/protoc-gen-doc /usr/bin/protoc-gen-doc
+RUN chmod +x /usr/bin/protoc-gen-grpc-java
 RUN chmod +x /usr/bin/protoc-gen-grpc-java
```

Comment:
duplicate?

Reply:
Correct. Should be other filename but since it works like that I will remove it